src/Traits/AliasesSelectionMethods.php

```diff
@@ -2,25 +2,30 @@
 
 namespace Nesk\Puphpeteer\Traits;
 
-/**
- * @method ElementHandle|null querySelector(string $selector)
- * @method array<int, ElementHandle> querySelectorAll(string $selector)
- * @method array<int, ElementHandle> querySelectorXPath(string $expression)
- */
```

Comment:
I'm not in favor of removing the doc blocks for native typings. The reason is because we can easily support a new Puppeteer version without having to edit the code.

Reply:
Hm. Good point, but:

1. this file don't covered by generator. 
2. i think all puppeteer versions have same syntax for this. its very basic actions.

Reply:
>     1. this file don't covered by generator.

Oh! That's right, my bad. 😅 

>     2. i think all puppeteer versions have same syntax for this. And i think it will remain. Its very basic actions.

Actually, we can't really know or be sure about that. They could perfectly add a second parameter to set options like extended filtering. I know this is not very likely to happen, but the goal here with PuPHPeteer is to act **as a simple proxy** and no more. If something new is added then I want to support it without having to update the code, otherwise it would require more maintainance (new issues, new PRs, new releases, etc.).

But maybe I'm missing something? Does using phpDocs **blocks** you with the way you use the PuPHPeteer? If so, explain me so I can understand and either provide you a solution or approve those changes.

Reply:
> But maybe I'm missing something? Does using phpDocs **blocks** you with the way you use the PuPHPeteer? If so, explain me so I can understand and either provide you a solution or approve those changes.

Well, its not blocking, but psalm syntax was not supported by my IDE. So there is no suggests and syntax checks. But if i manage to setup PHPStan to parse such phpDocs format in realtime then most of changes can be reverted. Its new feature in phpStorm